submissions/nadsatt/html-css-popup/style.css

```diff
@@ -0,0 +1,240 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+
+:root {
+    --list-padding: 12px;
+    --list-item-border-width: 1px;
+    --list-item-vertical-padding: 8px;
+    --list-item-content-height: 24px;
+    --popup-top-margin: 10px;
+    --color-black-light: #1A1A1A;
+    --color-black-dark: #000;
+    --color-black-transparent: rgba(0,0,0,0.41);
+    --color-rosy: rosybrown;
+    --color-gray-light: #ffffffc9;
+    --color-gray-middle: rgb(223, 223, 223);
+    --color-gray-dark: rgba(128, 128, 128, 0.829);
+    --color-white-light: #fff;
+    --color-white-dark: #ffffffef;
+}
+
+body {
+	font-family: 'Roboto', sans-serif;  
+	background-color: var(--color-black-light);
+} 
+
+* {
+	vertical-align: baseline;
+	font-weight: inherit;
+	font-family: inherit;
+	font-style: inherit;
+	font-size: 100%;
+	border: 0 none;
+	outline: 0;
+	padding: 0;
+    margin: 0;
+	outline: none;
+	list-style-type: none;
+	box-sizing: border-box;
+	text-decoration: none;
+}
+
+.nav {
+    width: 100vw;
+    height: auto;
+    background-color: var(--color-black-dark);
+    position: absolute;
+    top: 0;
+    left: 0;
+}
+
+.list {
+    width: 100%;
+    height: auto;
+    padding: var(--list-padding);
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content: flex-end;
+    align-items: center;
+}
+
+.list-item {
+    display: flex;
+    width: auto;
+    height: auto;
+    margin-left: 5px;
+    align-items: center;
+    position: relative;
+}
+
+.list-item__link {
+    width: auto;
+    height: calc(var(--list-item-content-height) + 
+            2 * var(--list-item-vertical-padding) + 
+            2 * var(--list-item-border-width));
+    line-height: var(--list-item-content-height);
+    padding: var(--list-item-vertical-padding) 7px;
+    border: var(--list-item-border-width) solid transparent; 
+    border-radius: 4px; 
+    color: var(--color-gray-light);
+    font-size: 18px;
+}
+
+.list-item__avatar-link {
+    padding: 3px;
+}
+
+.link__avatar {
+    border: 2px solid var(--color-gray-light);
+    border-radius: 50%;
+}
+
+.list-item__link .link__icon {
+    fill: var(--color-gray-light);
+}
+
+.list-item__checkbox {
```

Comment:
Your approach of hiding checkbox good but only for one situation:
Watch [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) how to hide checkbox in the right way